### PR TITLE
Typings for node-telegram-bot-api v6.0

### PR DESCRIPTION
**Pull request by konstantin24121**
Discussion: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60116
node-telegram-bot-api issue: https://github.com/yagop/node-telegram-bot-api/issues/965
node-telegram-bot-api PR with v6.0: https://github.com/yagop/node-telegram-bot-api/pull/966

* Add method *setChatMenuButton()*
* Add method *getChatMenuButton()*
* Add method *setMyDefaultAdministratorRights()*
* Add method *getMyDefaultAdministratorRights()*
* Add method *answerWebAppQuery()*
* Renamed the fields voice_chat_scheduled, voice_chat_started, voice_chat_ended, and voice_chat_participants_invited to video_chat_scheduled, video_chat_started, video_chat_ended, and video_chat_participants_invited

more changes synced with official changelog 

**Telegram Bot API 6.0 https://core.telegram.org/bots/api#april-16-2022**

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -40,42 +41,57 @@ declare namespace TelegramBot {
...
+        | 'video'
+        | 'video_note'
+        | 'voice'
+        | 'video_chat_started'
+        | 'video_chat_ended'
+        | 'video_chat_participants_invited'
+        | 'video_chat_scheduled'
+        | 'message_auto_delete_timer_changed'
+        | 'chat_invite_link'
+        | 'chat_member_updated';
```
Comment (daniel0611) [inline types/node-telegram-bot-api/index.d.ts:76]: This still needs the new [`web_app_data` message type](https://github.com/yagop/node-telegram-bot-api/compare/v0.56.0...72feca3d02293a3567ed927ce3792a33338da0eb#diff-a3e46f467e4f897823750e8a75b8e189d60e8ac5cc61ceaff63eac0e15f1c22fR58).

Comment (konstantin24121) [reply types/node-telegram-bot-api/index.d.ts]: Oh, thanks I missed it

**Review by hlxid — APPROVED** [review]
Code looks good to me :+1:, but still needs a review from an owner. It would be nice if you could update the tests as well.

**Comment by konstantin24121** [comment]
test added, hope they are ok, it's my first experience with typescript tests

**Review by hlxid — APPROVED** [review]
LGTM

**Comment by konstantin24121** [comment]
There are some button to start GH actions again? Tests runs fine locally, I confused with [Where is GH Actions?](https://github.com/DefinitelyTyped/DefinitelyTyped/labels/Where%20is%20GH%20Actions%3F) label

**Comment by daniel0611** [comment]
You can try amending your commit with no changes and force push it:
```shell
$ git commit --amend --no-edit
$ git push --force
```
That *should* retrigger GitHub Actions.
Or just add a new commit to this PR, that should work too.

**Review by hlxid — APPROVED** [review]

**Comment by konstantin24121** [comment]
@daniel0611 it works, but it need a review once again, annoying

**Comment by konstantin24121** [comment]
Ready to merge

**Pull request merged**